### ISSUE TITLE
Exporting Compiled Darknet

### ISSUE THREAD

**Issue by mon28** [AlexeyAB/darknet#6007]
Now that the darknet is successfully compiled using make, what was do we need to ship say on another system to be able to successfully run the common line commands for training and detection?

**Comment by WongKinYiu** [comment]
just copy all of needed dll and your compiled exe files.

**Comment by mon28** [comment]
I have compiled mine on a linux system. Could you list all the library files along with the darknet executable so that I don't miss anything out?

**Comment by WongKinYiu** [comment]
for linux, it need to copy many .o or .so files from your system, and all the files are depend on the version of library in your system.
I suggest you copy darknet folder to an another system and run, the error message will tell you what package you are missed.

following are some needed files of my case:
![image](https://user-images.githubusercontent.com/12152972/85103833-0631dd80-b23a-11ea-8c0b-ee4bf5e80212.png)
![image](https://user-images.githubusercontent.com/12152972/85103901-1d70cb00-b23a-11ea-8bec-43b3f5b993d0.png)

**Comment by mon28** [comment]
Yes, i had them in my obj folder with .o extension. Thanks for these. :-)
